ui/lang/is.js

```diff
@@ -0,0 +1,93 @@
+export default {
+  isoName: 'is',
+  nativeName: 'Íslenska',
+  label: {
+    clear: 'Hreinsa',
+    ok: 'Ókei',
+    cancel: 'Hætta við',
+    close: 'Loka',
+    set: 'Setja',
+    select: 'Velja',
+    reset: 'Endurstilla',
+    remove: 'Fjarlægja',
+    update: 'Uppfæra',
+    create: 'Búa til',
+    search: 'Leita',
+    filter: 'Sía',
+    refresh: 'Endurhlaða'
+  },
+  date: {
+    days: 'Sunnudagur_Mánudagur_Þriðjudagur_Miðvikudagur_Fimmtudagur_Föstudagur_Laugardagur'.split('_'),
+    daysShort: 'Sun_Mán_Þri_Mið_Fim_Fös_Lau'.split('_'),
+    months: 'Janúar_Febrúar_Mars_Apríl_Maí_Júní_Júlí_Ágúst_September_Október_Nóvember_Desember'.split('_'),
+    monthsShort: 'Jan_Feb_Mar_Apr_Maí_Jún_Júl_Ágú_Sep_Okt_Nóv_Des'.split('_'),
+    firstDayOfWeek: 0, // 0-6, 0 - Sunday, 1 Monday, ...
+    format24h: true,
+    pluralDay: 'dagar'
+  },
+  table: {
+    noData: 'Engin gögn tiltæk',
+    noResults: 'Engar samsvarandi skrár fundust',
+    loading: 'Augnablik...',
+    selectedRecords: function (rows) {
+      return rows === 1
+        ? '1 færsla valin.'
+        : (rows === 0 ? 'No' : rows) + ' færslur valdar.'
```

Comment:
Shouldn't the 'No' here be: "Engar"? As in "Engar færslur valdar"?

Reply:
Hot damn, good catch! Missed the inline if there. Correct on the translation, I'll update.